github/pulls_reviews.go

```diff
@@ -183,6 +218,15 @@ func (s *PullRequestsService) CreateReview(ctx context.Context, owner, repo stri
 		return nil, nil, err
 	}
 
+	// Detect which style of review comment is being used.
+	if isCF, err := review.isComfortFadePreview(); err != nil {
+		return nil, nil, err
+	} else if isCF {
+		// If the review comments are using the comfort fade preview fields,
+		// then pass the comfort fade header.
+		req.Header.Set("Accept", "application/vnd.github.comfort-fade-preview+json")
```

Comment:
Could you please change this to the already-defined constant: `mediaTypeMultiLineCommentsPreview` (in github/github.go)?

Reply:
done